### DISCUSSION TITLE
Null Object Reference

### DISCUSSION THREAD

**Discussion by Dancetrap** [FunkinCrew/Funkin#1910] [Q&A]
Hi.
So I'm trying to boot up the version of the game that's used for modding, I follow the directions on ninjamuffin's page, but everytime I type in "lime test windows -debug" on the console, I get this error message that says "Null Object Reference." And I don't understand what's going wrong. Am I missing something? Because I didn't change anything else in the mod, I'm just trying to load it up for the first time.
Thank you for your help.

**Comment by libauror-a** [comment]
uhhh build log?

**Comment by SilverT4** [comment]
Do you have any console logs? What appears in the console when the Null Object Reference error appears?

**Comment by Dancetrap** [comment]
Nothin much. But I did figure it out. But thanks for the help.

**Reply by ghost** [reply]
> Nothin much. But I did figure it out. But thanks for the help.

can you post the solution?
